### PR TITLE
Sync docs from 6.2.0

### PR DESCRIPTION
**Pull request by kezhenxu94**
main repo sha: 11e7cd44f7f9d1c4aef51036d2391c302a981b33

### CODE REVIEW

**Comment by wu-sheng** [comment]
Merge it if you think it is ready. This is nearly impossible to review :)

**Comment by wu-sheng** [comment]
I would suggest separating copy and update by two PR, then the reviewer knows the changes.

File: docs/zh/6.2.0/FAQ/EnhanceRequireObjectCache-Cast-Exception.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.shouldApplyTo(AbstractHandlerMethodExceptionResolver.java:47)
+	at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:131)
+	at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:76)
+	...
+```
+
+### 原因
+此异常可能由某些热部署工具(`spring-boot-devtool`)或者其它可以在运行时改变`classloader` 的工具导致。
+### 解决方法
+1. 生产环境不会出现此错误是因为开发者工具自动关闭了，详细见[spring-boot-devtools说明](https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-devtools.html)
```
Comment (SataQiu) [inline docs/zh/6.2.0/FAQ/EnhanceRequireObjectCache-Cast-Exception.md]:
```suggestion
1. 生产环境不会出现此错误是因为开发者工具被自动关闭了，详细见[spring-boot-devtools说明](https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-devtools.html)
```

Comment (SataQiu) [reply docs/zh/6.2.0/FAQ/EnhanceRequireObjectCache-Cast-Exception.md]: Too many files, hard to review. :worried:

Comment (kezhenxu94) [reply docs/zh/6.2.0/FAQ/EnhanceRequireObjectCache-Cast-Exception.md]: yes，I'll separate update and 6.2.0 to two commits

**Comment by kezhenxu94** [comment]
Update commit and copy commit are seperated, I'll merge this and if @SataQiu you find anything can be optimized, feel free to open another PR, thanks

**Comment by wu-sheng** [comment]
I opened  sync to master option of this repo.

**Comment by wu-sheng** [comment]
Also, I think you still could merge as you are admin, right?

**Comment by kezhenxu94** [comment]
I'll seperate this PR into two because I can only two commits

> Also, I think you still could merge as you are admin, right?

Yes, and I also enabled "create a merge commit" button to keep the commit history when really needed, such as this kind of PR

**Pull request merged**